sdk/appconfiguration/app-configuration/src/generated/src/appConfigurationContext.ts

```diff
@@ -12,7 +12,7 @@ import * as coreHttp from "@azure/core-http";
 import * as Models from "./models";
 
 const packageName = "app-configuration";
-export const packageVersion = "1.0.0-preview.9";
+export const packageVersion = "1.0.0-preview.10";
```

Comment:
This doesn't have to be addressed here but this makes me thing we should have something like a 'release_checklist.md` file in the package that includes checking/updating this const.

Reply:
We have a unit test that fails when it's not updated but I admit it could include some more commentary on what it affects.

https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/appconfiguration/app-configuration/test/package.spec.ts

Reply:
Added that little bit of commentary :)

Reply:
Oh cool, didn't realize there was already a test for this. That's great :)